hw02/hw02_exploring_gapminder.Rmd

```diff
@@ -0,0 +1,65 @@
+library(gapminder)
+library(ggplot2)
+library(palmerpenguins)
+library(tidyverse)
+library(dplyr)
+
+# 1.1 & 1.2
+
+gapminder %>%
+  select(country, year, gdpPercap) %>%
+  filter(country %in% c("Peru" , "Japan", "Canada"), year>1969 & year<1980)
+
+# 1.3
+
+gapminder %>%
+  mutate(lifeExpDrop = lag(lifeExp)
+
+# 1.4 
+
+gapminder %>%
+  select(country, gdpPercap) %>%
+  mutate(maxgdpPercap = max(gdpPercap)) %>%
+  group_by(country)
+
+# 1.5
+
+gapminder %>%
+  filter(country == "Canada") %>%
+  select(country, lifeExp, gdpPercap) %>%
+  ggplot() + aes(x=lifeExp, y=gdpPercap) + geom_point() +
+  scale_y_continuous(trans = "log10")
+  
+# 2.1
+
+penguins %>%
+  summarize(
+  across(c(bill_length_mm, bill_depth_mm),
+  list(mean = ~ mean(.x, na.rm = TRUE),
+      sd = ~ sd(.x, na.rm = TRUE),
+      min = ~ min(.x, na.rm = TRUE),
+      max = ~ max(.x, na.rm = TRUE))))
+      
+penguins %>%       
+  ggplot() + 
+      aes(x = bill_length_mm, y = bill_depth_mm) + 
+      geom_point(fill = "blue", color = "brown", alpha = .5, size =3) +
```

Comment:
It seems like the fill is still brown when running this. Overall it looks good though